homework-solutions/js/hw-5/task-1.js

```diff
@@ -18,13 +27,29 @@ let evenNumbersResult = '';
  */
 
 let smilePatternResult = '';
+for (let i = 1; i <= 5; i++){
+    let line = '';
+    for (let j = 1; j <= i; j++){
+        line += ':)';
+    }
+    smilePatternResult += line;
```

Comment:
Как вариант без добавления промежуточной переменной
```suggestion
  for (let j = 1; j <= i; j++) {
    smilePatternResult += ':)';
  }
```